inference/nlp/realtime/llm/bloom_176b/djl_deepspeed_deploy.ipynb

```diff
@@ -0,0 +1,613 @@
+{
```

Comment:
<p>Missing a period.</p><p>Is passing a Text preferred over saying you're passing a string?</p><p>What about discussing these parameters?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>added explanations and also added links for further read</p>